contracts/coordinator/src/msg.rs

```diff
@@ -25,4 +26,10 @@ pub enum ExecuteMsg {
 pub enum QueryMsg {
     #[returns(WorkerSet)]
     GetActiveVerifiers { chain_name: ChainName },
+
+    #[returns(bool)]
+    CheckWorkerCanUnbond {
+        worker_address: Addr,
+        chains: HashSet<ChainName>,
```

Comment:
actually I realized we are not tracking the chains associated with a given worker in this contract. But didn't we want to do that? Wasn't that the whole reason to move to a push model and actually store the active workersets in this contract? Otherwise, we already have the prover address, so why not just query the prover directly each time.

Reply:
agreed, this check should be very simple: "is this worker ready to unbond?" the coordinator should figure out all the necessary information

Reply:
Thanks for mentioning it. This is fixed now.